pages/glossary/owner-element.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Owner element
+key: owner-element
+---
+
+
+If the node is referenced by an element through the `aria-owns` attribute, then the owner element is the first node in the DOM tree that references it through `aria-owns`.
```

Comment:
@kasperisager found this case that is not covered by the current definition:

`<html>
<body>
<div aria-owns="child">
Parent
</div>
<div id="child">
Child 1
</div>
<div id="child">
Child 2
</div>
</body>
</html>
`
Parent owns Child 1
Parent does not own Child 2